helm/polaris/templates/job.yaml

```diff
@@ -0,0 +1,87 @@
+{{- if eq .Values.polarisServerConfig.metaStoreManager.type "eclipse-link" }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "polaris.fullname" . }}
+  labels:
+    {{- include "polaris.labels" . | nindent 4 }}
+    {{- if .Values.podLabels }}
+    {{- tpl (toYaml .Values.podLabels) . | nindent 4 }}
+    {{- end }}
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  template:
+    metadata:
+      annotations:
+        {{- if .Values.podAnnotations }}
+          {{- tpl (toYaml .Values.podAnnotations) . | nindent 8 }}
+        {{- end }}
+      labels:
+        {{- include "polaris.selectorLabels" . | nindent 8 }}
+        {{- if .Values.podLabels }}
+        {{- tpl (toYaml .Values.podLabels) . | nindent 8 }}
+        {{- end }}
+    spec:
+      {{- if .Values.persistenceConfigSecret }}
+      initContainers:
+        - name: init-config
+          image: registry.access.redhat.com/ubi9/openjdk-21:1.20-2.1721752936
```

Comment:
Maybe move this image reference to `values.yaml`?

Reply:
> Maybe move this image reference to `values.yaml`?

I like this. Added. Thanks for review.